src/axolotl/core/trainer_builder.py

```diff
@@ -1428,6 +1428,11 @@ def build(self, total_num_steps):
         )
 
         training_arguments_kwargs = {}
+
+        training_arguments_kwargs[
+            "include_tokens_per_second"
+        ] = self.cfg.include_tokens_per_second
```

Comment:
```suggestion
        if self.cfg.include_tokens_per_second is not None:
            training_arguments_kwargs[
                "include_tokens_per_second"
            ] = self.cfg.include_tokens_per_second
```

The trainer expects boolean with False being default. Since we default to None, maybe this is better? (spacing in edit might be off a bit)